base_geoengine/geo_model.py

```diff
@@ -178,7 +163,10 @@ def get_edit_info_for_geo_column(self, cursor, uid, column, context=None):
         res['edit_raster'] = raster_obj.read(
             cursor, uid, raster_id[0], context=context)
         res['geo_type'] = field.geo_type
-        res['srid'] = field.srid
+        # get srid from _columns as srid is wrong in _fields
+        res['srid'] = self._columns.get(column)._srid
```

Comment:
@yvaucher see https://travis-ci.org/OCA/geospatial/jobs/120637898#L481 That's why you can't get the srid from the column...